sql-plugin/src/main/scala/com/nvidia/spark/rapids/RegexParser.scala

```diff
@@ -730,6 +747,8 @@ sealed case class QuantifierVariableLength(minLength: Int, maxLength: Option[Int
 
 sealed trait RegexCharacterClassComponent extends RegexAST
 
+// TODO: Consider merging some of these subclasses to handling escaping
```

Comment:
Is this TODO needed?  Are we doing to do something with it? Is there a follow on issue that needs to be filed?

Reply:
oops, forgot to remove an old comment, fixed